ligo/test/baseDAO-ligo-tests.cabal

```
@@ -1,8 +1,10 @@
 cabal-version: 2.2
 
--- This file has been generated from package.yaml by hpack version 0.34.2.
+-- This file has been generated from package.yaml by hpack version 0.33.0.
 --
 -- see: https://github.com/sol/hpack
+--
+-- hash: e553a222f27a7fa6dd2e3a6ef625022c2e2c14fe13f83cc6bca9d5aafc19547f
```

```suggestion
-- This file has been generated from package.yaml by hpack version 0.34.2.
--
-- see: https://github.com/sol/hpack
```
same as for the other one

---

Yeah, I forgot about this. Usually, I have to run `./scripts/generate-cabal-files.sh` and remove the hash manually.

---

To be sure you could also run the script with nix and it should always produce the right resulr:
```
nix run -f ci.nix stack2cabal pkgs.diffutils -c ./scripts/generate-cabal-files.sh
```